rtv/utils.py

```diff
@@ -16,6 +16,27 @@
 
 ESCAPE = 27
 
+help_msg = """Global Commands
+  Arrow Keys or `hjkl`: Navigations.
+  `r` or `F5`         : Refresh the current page.
+  `q`                 : Quit the program.
+  `o`                 : Open the url of the selected item in the default web
+                        browser.
+  `?`                 : Show this help message.
+
+Subreddit Mode
+  Right or `Enter`    : Open the currently selected submission in a new page.
+  `/`                 : Open a prompt to switch to a different subreddit.
+
+Submission Mode
+  Right or `Enter`    : Toggle the currently selected comment between hidden
+                        and visible.
+  Left                : Exit the submission page and return to the subreddit.
+"""
+
+help_msg = help_msg.split("\n")
```

Comment:
I feel like this should be inside of a function. Can be as simple as
```python
HELP = """blah blah blah"""

def display_help(stdscr):
    message = HELP.split("\n")
    display_message(stdscr, message)
```
That way you don't have to import the raw string inside of subreddit.py and submission.py